packages/page-bounties/src/Bounty.tsx

```diff
@@ -55,144 +58,254 @@ function Bounty ({ bestNumber, bounty, className = '', description, index, propo
       <tr className={className}>
         <td>
           <BountyStatusView
+            blocksUntilPayout={blocksUntilPayout}
             bountyStatus={bountyStatus}
             proposals={proposals}
             status={status}
           />
         </td>
-        <td data-testid='description'>{truncateTitle(description, 30)}</td>
+        <td
+          className='description-column'
+          colSpan={2}
+          data-testid='description'
+        >
+          <div title={description}>
+            {description}
+          </div>
+        </td>
         <td><FormatBalance value={value} /></td>
-        <td>{curator ? <AddressSmall value={curator} /> : EMPTY_CELL}</td>
-        <td><DueBlocks dueBlocks={blocksUntilUpdate} /></td>
-        <td>{beneficiary ? <AddressSmall value={beneficiary} /> : EMPTY_CELL}</td>
-        <td><DueBlocks dueBlocks={blocksUntilPayout} /></td>
+        <td>{curator && <AddressSmall value={curator} />}</td>
         <td>
-          <BountyActions
-            bestNumber={bestNumber}
-            description={description}
-            index={index}
-            proposals={proposals}
-            status={status}
-            value={value}
-          />
+          {blocksUntilPayout
+            ? <DueBlocks
+              dueBlocks={blocksUntilPayout}
+              until={'payout'}
+            />
+            : ''}
+          {blocksUntilUpdate
+            ? <DueBlocks
+              dueBlocks={blocksUntilUpdate}
+              until={'update'}/>
```

Comment:
```suggestion
          {blocksUntilUpdate
            ? <DueBlocks
              dueBlocks={blocksUntilUpdate}
              until={'update'}
            />
```

Consistency with the block just above and elsewhere.